packages/artillery-plugin-apdex/index.js

```diff
@@ -36,13 +36,21 @@ class ApdexPlugin {
 
       function apdexAfterResponse(req, res, userContext, events, done) {
         const total = res.timings.phases.total;
+        const counts = {
```

Comment:
Personal preference, but I'd emit all 3 metrics with value of `0` here at the beginning of the function, and keep the rest of the function the same.